tests/Command/SellerOnboardingCommandTest.php

```diff
@@ -159,4 +159,24 @@ public function testShopUpdateDateCheckpoint()
         $this->executeCommand();
         $this->assertEquals(MiraklMock::SHOP_DATE_1_EXISTING_WITH_OAUTH_URL, $this->configService->getSellerOnboardingCheckpoint());
     }
+
+    public function testIgnoredNewShop()
+    {
+        $this->deleteAllAccountMappingsFromRepository();
+//        $this->mockAccountMapping(MiraklMock::SH, StripeMock::ACCOUNT_BASIC, false, false, null, true);
```

Comment:
Was it intended to leave this commented out? tests/Command/SellerOnboardingCommandTest.php LN: 166